xformers/components/residual.py

```diff
@@ -29,6 +29,27 @@ class LayerNormStyle(str, Enum):
     DeepNorm = "deepnorm"
 
 
+class NormalizationType(str, Enum):
+    LayerNorm = "layernorm"
+    Skip = "skip"
+    # TODO: BatchNorm = "batchnorm"
```

Comment:
this is probably to be done in another PR, requires deferred init or some clever way to get the embedding size at that point, so for now just handle the "no normalization" path